src/test/java/org/springframework/cloud/bindings/boot/TestHelper.java

```diff
@@ -0,0 +1,17 @@
+package org.springframework.cloud.bindings.boot;
```

Comment:
you can simply move this static method to your test class `EurekaBindingsPropertiesProcessorTest`

Reply:
Hmm. This helper code is also used by a few different test classes in the pem package, so it seemed a bit messy to have a bunch of different test classes all calling a static method on one test class. We put it into a helper class to make it clearer that it is shared code. Is there a better pattern?

Reply:
well, that's fine, it's just for a test. When I wrote this I probably did not pay attention to the `pem` package usage.

You can leave as is.